garageScript/c0d3-app
# feat(setting): Create linked accounts section

## Motivation

Adding the 3rd section, linked accounts to the settings page.

## Changes

- Create linked accounts section
- Test the code to ensure it's working as expected
- Add styles to make the list look nicer
- Replace `GetApp` query with `UserInfo` as `UserInfo` has extra fields that we need

## Testing

[Preview link](https://c0d3-app-git-fork-flacial-2705-cors-for-reques-124d7b-c0d3-prod.vercel.app/)

0. Login
1. Go to `/settings/account`
2. Scroll down to `Linked accounts` section
3. Try to connect to Discord and ensure it's working
4. Try to unlink from Discord and ensure it's working

## Related issues

- #1629
